### DISCUSSION TITLE
All PRs are closed in the Github page

### DISCUSSION THREAD

**Discussion by JackyWoo** [facebookincubator/velox#11090] [Q&A] [answered]
I am new to velox and I found all PRs are closed in the Github page. 

May I ask what is the open source workflow in velox? If a person is not meta employee whether he can review code?

Thanks.

**Comment by kgpai** [comment, marked as answer]
Hi Jacky, 

>  I found all PRs are closed in the Github page.
PR's are open on the github page : https://github.com/facebookincubator/velox/pulls . 


> If a person is not meta employee whether he can review code? 

Yes , there are many non meta employees who review code and you are free to review any PR. We welcome any contribution but please read up on the [Contribution guide](https://github.com/facebookincubator/velox/blob/main/CONTRIBUTING.md) first .

**Comment by JackyWoo** [comment]
@kgpai may I ask why all PRs are closed in the Github?

**Reply by kgpai** [reply]
Not sure I understand what you mean here ?

**Reply by JackyWoo** [reply]
Thanks for your reply. 

I mean in most of Github projects when a PR is merged into main branch the PR is marked as `merged` but in velox the status of merged PR is `closed`.
